src/Navigation/Navigation.js

```diff
@@ -0,0 +1,23 @@
+import "./Navigation.css";
+import { useNavigate } from "react-router-dom";
+
+export const Navigation = ({loggedIn, setLoggedIn}) => {
+  const navigate = useNavigate();
+  const handleLoginNavigate = () => {
+    setLoggedIn(false);
+    navigate(`/login`);
+  }
+  const visitSaved = () => {
+    navigate(`/saved`);
```

Comment:
really good work on this, depending on if ed utilizes auth0 for his login on the backend this will make it much easier to code